insta_scrape.py

```diff
@@ -0,0 +1,70 @@
+from selenium.webdriver.common.by import By
+import time
+
+waiting_seconds = 4
+
+
+def handle_cookiebanner(driver, allow_cookies=1):
+    print('handle cookie banner..')
+    time.sleep(waiting_seconds)
+    button_class = '_a9--'
+    allow_cookies_button_class = '_a9_0'
+    decline_cookies_button_class = '_a9_1'
```

Comment:
warum sind die classes so weird?

Reply:
The CSS classes in instagram are completley obfuscated. These classes are CSS classes and are used further down to select the right HTML element.